invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -4,6 +4,8 @@
 
 @import "../../landing_page/creatibutors.less";
 @import "../../landing_page/licenses.less";
+@import "site.variables";
```

Comment:
Importing the site variables here is incorrect, it will break the theming

Reply:
my bad! will remove it